### PR TITLE
Copy to clipboard button for raw xml

### PR DESCRIPTION
**Pull request by benrudolph**
@proteusvacuum http://manage.dimagi.com/default.asp?243589

cc: @emord 
copy xml

### CODE REVIEW

**Comment by proteusvacuum** [comment]
😍 
Those buttons are really prominent for something that I imagine is used almost never, but as long as I can double click in that area and it does what I expect, this is awesome.

File: corehq/apps/reports/templatetags/xform_tags.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def render_form_xml(form):
     if isinstance(xml, unicode):
         xml.encode('utf-8', errors='replace')
     formatted_xml = indent_xml(xml) if xml else ''
-    return '<pre class="fancy-code prettyprint linenums"><code class="no-border language-xml">%s</code></pre>' \
```
Comment (proteusvacuum) [inline corehq/apps/reports/templatetags/xform_tags.py]: I think this is the only instance of this anywhere in the codebase... which means I think we can remove all the `{% include "imports/fancy-code.html" %}` everywhere, and also the dependency on `jQuery.selectText`. Hooray!

**Review by emord — APPROVED** [review]
Maybe one day we can use codemirror :)

File: corehq/apps/reports/templatetags/xform_tags.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/reports/templatetags/xform_tags.py]: nice, ok i can add that in

**Review by proteusvacuum — APPROVED** [review]
☠️

**Comment by czue** [comment]
:high_heel: :construction_worker_man:

File: corehq/apps/hqwebapp/templates/imports/fancy-code.html [hunk 2]
```diff
@@ -1,26 +0,0 @@
-{% load hq_shared_tags %}
-{% include "imports/bootstrap.google-prettify.html" %}
```
Comment (czue) [inline corehq/apps/hqwebapp/templates/imports/fancy-code.html]: I think this file can also be deleted now

**Review by czue — APPROVED** [review]
love this. :skull:

**Pull request merged**